doc_admin/AdminManual.adoc

```diff
@@ -57,11 +61,11 @@ The aim of this program is to manage, in the simplest manner, the hospital admin
 
 Open Hospital is a stand-alone Javafootnote:[Java is a registered trademark of Oracle and/or its affiliates.] software program that does not require installation and is often known as a portable installation. The .zip file can be copied and moved everywhere on the target system, whether the operating system is Windows or Linux.
 
-The software acts as a client application. The server part utilizes a DBMS (Database Management System) to persist information. From the inception of the software, the free and open-source MySQLfootnode:[See https://www.mysql.com/] database system is the DBMS of choice.
+The software acts as a client application. The server part utilizes a DataBase Management System (DBMS) to archive, share and present the information. The free and open-source MySQLfootnode:[See https://www.mariadb.org] MariaDB is the database server of choice.
 
-The real part of the installation includes installing the JRE (Java Runtime Environment), installing the MySQL Server, and the creation of the database (DB). This installation is accomplished through the execution of a script in the folder sql/ within the .zip file folders.
+The procedure includes the installation the JRE (Java Runtime Environment), the MySQL Server and the creation of the database through the execution of a script in the folder sql/ within the .zip file folders.
```

Comment:
`The procedure includes the installation the JRE` should be `The procedure includes the installation of the JRE`

Reply:
fixed